crates/rustc_codegen_spirv/src/spirv_type_constraints.rs

```diff
@@ -727,15 +727,15 @@ pub fn instruction_signatures(op: Op) -> Option<&'static [InstSig<'static>]> {
         | Op::ExecuteCallableKHR
         | Op::ConvertUToAccelerationStructureKHR
         | Op::IgnoreIntersectionKHR
-        | Op::TerminateRayKHR => reserved!(SPV_KHR_ray_tracing),
+        | Op::TerminateRayKHR => {}
```

Comment:
Can you change these to be like the other ones which have a comment like this:
```rust
// NOTE(eddyb) we actually use these despite not being in the standard yet.
```
(and the `reserved!` being commented out, not removed)

The reason I want to keep this structure is that the spec still lists these instructions are reserved, and I wanted to be able to track changes to the spec long-term, by mirroring its order and categorization.